### PR TITLE
🐍 Fix `no-underscore-dangle` eslint errors

### PR DESCRIPTION
**Pull request by mihaerzen**

### CODE REVIEW

**Review by jalezi — APPROVED** [review]
Thanks.

File: src/components/Shared/Leaflet/Markers.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ import iconShadow from 'leaflet/dist/images/marker-shadow.png';
 
 const { GEO_LOCATION } = MAP;
 
+// TODO: Why is this deleted?
```
Comment (stefanb) [inline src/components/Shared/Leaflet/Markers.js:11]: Maybe it doesn't have to be deleted, I saw it in some sample, and doesn't make much sense, but it worked and I just left it there.

**Review by stefanb — APPROVED** [review]

File: src/components/Shared/Leaflet/Markers.js (hunk 1 above)
Comment (mihaerzen) [reply src/components/Shared/Leaflet/Markers.js:11]: It broke eslint and I had to specifically ignore the rule for this line. I would vote to remove all three of these lines actually. What do you think?

Comment (stefanb) [reply src/components/Shared/Leaflet/Markers.js:11]: I tried removing that line, and it left the map on doctor card with missing marker: 
image
so i guess it is still needed :)

Comment (stefanb) [reply src/components/Shared/Leaflet/Markers.js:11]: ...and the name starting with underscore out of our control, making the eslint rule disabling the only option.

**Comment by mihaerzen** [comment]
Go ahead and merge this @stefanb or @jalezi . I have resolved the questions I had.

**Comment by stefanb** [comment]
tnx, merged!

**Pull request merged**